config-gen/templates/patches/cert-manager/annotation.template.yaml

```diff
@@ -0,0 +1,5 @@
+{{ if .Spec.Enabled "cert-manager" }}
```

Comment:
How we will ensure that KB boilerplates and the templates here are doing the same? 
Could be possible we centralize the implementation?

Reply:
I think in many cases these are similar but not the same as the KB boilerplate templates.  The templates generate Kustomize files, including kustomization.yamls and patch files.  It might be possible to refactor the boilerplate templates to be reusable from both contexts.